### DISCUSSION TITLE
How can i config some item backgroup color

### DISCUSSION THREAD

**Discussion by hard-game** [cncf/landscape2#764] [Q&A] [answered]
can i change some items backgroup color?

**Comment by tegioz** [comment, marked as answer]
Hi @hard-game 👋

No, not really. The closest we have is the [`featured items` feature](https://github.com/cncf/landscape2/blob/dbb790c519b5a44dab78710546e2fdb252f66bc5/docs/config/settings.yml#L121-L155), but it's not the same.

**Reply by hard-game** [reply]
Thanks
